docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -15,12 +15,13 @@ which edx-proctoring can call directly since it is a plugin that is installed di
 
 Decision
 --------
-We've decided to use an event driven architecture because unlike the legacy system (edx-porctoring), edx-exams is an independent service.
+We've decided to use an event driven architecture because unlike the legacy system (edx-proctoring), edx-exams is an independent service.
 As such, we plan to use the event bus to send info to edx-platform services without needing a response as one would in a REST framework.
 
-Below are lists of downstream effects of exam submission and review that we will or will not be translating into events in edx-exams as part of this decision.
+Below are lists of downstream effects of exam submission and review that we will or will not be impementing as downstream effects that will be
```

Comment:
```suggestion
Below are lists of downstream effects of exam submission and review that we will or will not be implementing as downstream effects that will be
```

Reply:
I'm having a hard time understanding this sentence. I've read it three times and am still struggling. Maybe it's because it's Friday afternoon.

Are you trying to say something like this?

"Below are lists of expected downstream effects of exam submission and review. We have decided to implement some downstream effects and not others; these decisions are enumerated below. Those that will be implemented will be triggered by events emitted by the `edx-exams` service, as described in this decision."